src/component/badge/style/_scheme.scss

```
@@ -0,0 +1,43 @@
+////
+/// Badge Module
+/// @group badge
+////
+
+@mixin _badge-scheme($legacy: false) {
+  #{ns(badge)} {
+    // @include scheme-element-text-color(bf500-plain, true);
+    // @include scheme-element-bg-color(bf200-bf300, true);
+    @include scheme-element-text-color(default neutral, $legacy);
```

mettre à jour avec le clean couleur : 
```suggestion
    @include scheme-element-text-color(default neutral, (legacy: $legacy));
```

---

OK + neutral devient grey